src/main/deltachat.js

```diff
@@ -145,6 +145,17 @@ class DeltaChatController {
     if (typeof this._render === 'function') this._render()
   }
 
+  /**
+   * Dispatched when showing the AboutDialog (owned by ScreenController)
```

Comment:
It has to be `ScreenController` that owns this, if we want to show it without being logged in.

Reply:
Yeah I have an idea on how to refactor this with a sort of `DialogController`, can take a look.